docs/01-getting-started.md

```diff
@@ -6,14 +6,23 @@ a Sleeper instance with a simple schema, and writes some random data into a tabl
 scripts to see how much data is in the system, run some example queries, and view logs to help understand what the system is
 doing. It is best to do this from an EC2 instance as a significant amount of code needs to be uploaded to AWS.
 
-Before running this demo functionality, you will need the following intalled (see the [deployment guide](02-deployment-guide)
+Before running this demo functionality, you will need the following intalled (see the [deployment guide](02-deployment-guide.md)
 for more information on getting set up correctly) and you will need your CLI to be logged into your AWS account:
 
-- Maven
-- The AWS CLI
-- CDK
-- Docker
-- Java
+* [AWS CDK](https://docs.aws.amazon.com/cdk/latest/guide/cli.html)
+* [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html): v2
+* [Bash](https://www.gnu.org/software/bash/): At least version 4: use `bash --version`.  The default version on a Mac is 3.2, which is too old.
```

Comment:
Can we note here that this issue will fix this so that the standard version of bash on Mac will work https://github.com/gchq/sleeper/issues/36 ?

Reply:
Great, that sounds really good.  Happy to remove once issue #36 is merged (also happy to look at that issue).